code/datums/setup_option/backgrounds/profession.dm

```diff
@@ -0,0 +1,143 @@
+//Where we worked our entire life or perhaps not even that, what existence we went throught
+/datum/category_group/setup_option_category/background/profession
+	name = "Profession"
+	category_item_type = /datum/category_item/setup_option/background/profession
+
+/datum/category_item/setup_option/background/profession
+
+/*
+	With professions (formerly carreers this just sounds better) due to overall reduction in points gained should go way more ham
+	general rule of thumb should be, compensate those that give perks with way less stats gained than those that don't IF you even
+	give away perks to make it more attractive to some people but to others just who cares.
+	Without perk it should amount to around 25-30 points
+	With perk keep the minuses at around 10-15 in the minus or keep the balance equilibrium at least even
+*/
+
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+/////////////
+/* General */
+/////////////
+/datum/category_item/setup_option/background/profession/jack
+	name = "Jack-of-all-trades"
+	desc = "Average Joe, good for nothing, never learned anything useful, all of these describe you perfectly. \
```

Comment:
wants quotes around "Average Joe", "good for nothing", and "never learned anything useful", and then a dash after useful"